inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.dao.entity;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+@Builder
+public class SortIdParamsEntity implements Serializable {
```

Comment:
Wouldn't it be better to change the entity name to SortParam?
This table not only stores the ID of the Sort but can also be extended to store other parameters of the Sort.

BTW, the table name already has a plural meaning, so there is no need to use the plural form. 
For example, all user information is stored in the `user` table, and the table name is called `user`, not `users`.